mapping_definitions/deputy_daytime_phonenumbers_mapping.json

```diff
@@ -4,7 +4,7 @@
             "is_complete": true
         },
         "sirius_details": {
-            "is_pk": true,
+            "is_pk": "",
```

Comment:
This is actually the pk, why remove it? This field is used to calculate ids in the `integration` stage